### DISCUSSION TITLE
Jekyll Paginate V2 Error

### DISCUSSION THREAD

**Discussion by me-nkr** [mmistakes/minimal-mistakes#3667] [General]
I have been trying Jekyll paginate v2 for some time, but I am receiving an error.

When the posts are paginated the `href` value for pages is wrong.

For example, the button for the 2nd page takes me to `baseurl/page2` instead of `baseurl/category/page/2`.

If anyone tried the plugin, faced the issue, and solved it...

Please let me know how to solve it.

Thanks in advance.

**Comment by iBug** [comment]
This theme doesn't come with stock support of Paginate V2. Try #2636 instead.

**Reply by me-nkr** [reply]
I read through the PR comments and understood nothing much.

As much as I understood, you also had the issue of the wrong URL situation. And by visiting your blog, I can see the issue has been resolved for you.

So did you write a fix and minimal mistakes merge it?

**Reply by iBug** [reply]
You need to take that PR's content and try to apply it to your website repository. It's a full implementation but Michael hasn't merged that it.
